tests/onnx/test_onnx_model_export.py

```diff
@@ -346,10 +341,17 @@ def test_model_save_load_multiple_inputs(onnx_model_multiple_inputs_float64, mod
     assert onnx.checker.check_model.called
 
 
+@pytest.mark.parametrize("save_as_external_data", [True, False])
```

Comment:
@BenWilson2 will this do?
<img width="640" alt="Screenshot 2023-10-27 at 8 49 27 AM" src="https://github.com/mlflow/mlflow/assets/148037680/79648836-4174-447d-a328-ad10f9b5d400">

Reply:
Perfect! :)